ui/src/containers/ClusterMonitoring.js

```diff
@@ -275,12 +305,14 @@ const makeClusterStatus = (state, props) => {
     value = CLUSTER_STATUS_UP;
     label = intl.messages.cluster_up_and_running;
   }
-
   if (cluster.error) {
     value = CLUSTER_STATUS_DOWN;
     label = intl.messages[cluster.error] || cluster.error;
   }
-
+  if (!cluster.isPrometheusVolumeProvisioned) {
```

Comment:
Almost, that should be:

```suggestion
  if (!state.app.monitoring.isPrometheusApiUp) {
```